.github/workflows/devcontainer.yml

```diff
@@ -142,6 +139,7 @@ jobs:
       timeout-minutes: 3
       run: |
         set -x
+        git --no-pager diff --exit-code
```

Comment:
There shouldn't be any local changes to the repo - if there are it can cause cache reuse problems.